catalog-rest-service/src/main/resources/json/schema/entity/services/connections/dashboard/tableauConnection.json

```diff
@@ -47,6 +47,11 @@
       "description": "Tableau Site Name.",
       "type": "string"
     },
+    "siteUrl": {
+      "title": "Site Url",
+      "description": "Tableau Site Url.",
+      "type": "string"
```

Comment:
do we need to use the URI format for this URL?

Reply:
It won't be needed here since this Url can be just a normal string